api/src/main/java/api/QuestionResource.java

```diff
@@ -33,14 +33,44 @@
     Observable<Question> getQuestion(Auth auth, @PathParam("questionId") long questionId);
 
     /**
-     * Return a list of latest questions with a limit
+     * Return a list of latest questions
      *
      * @param limit
      * @return questions
      */
     @GET
     @Path("questions/latest")
-    Observable<List<Question>> getLatestQuestion(@QueryParam("limit") Integer limit);
+    Observable<List<Question>> getLatestQuestions(@QueryParam("limit") Integer limit);
+
+    /**
+     * Return a list of the highest voted questions
+     *
+     * @param limit
+     * @return questions
+     */
+    @GET
+    @Path("questions/popular")
+    Observable<List<Question>> getPopularQuestions(@QueryParam("limit") Integer limit);
+
+    /**
+     * Return a list of the highest voted questions without any answers
+     *
+     * @param limit
+     * @return questions
+     */
+    @GET
+    @Path("questions/popularunanswered")
+    Observable<List<Question>> getPopularUnansweredQuestions(@QueryParam("limit") Integer limit);
```

Comment:
Maybe we could just add another query-parameter to  the questions/popular enpoint specifying whether it should be answered or not?

Reply:
I'm not sure. "popular" includes both answered and unanswered, so the param would have to be something like "onlyUnanswered".